django_opentracing/tracing.py

```diff
@@ -126,10 +126,9 @@ def _finish_tracing(self, request, response=None, error=None):
 
         if error is not None:
             scope.span.set_tag(tags.ERROR, True)
-            scope.span.log_kv({
-                'event': tags.ERROR,
-                'error.object': error,
-            })
+            scope.span.set_tag('sfx.error.message', str(error))
+            scope.span.set_tag('sfx.error.object', str(error.__class__))
+            scope.span.set_tag('sfx.error.kind', error.__class__.__name__)
```

Comment:
I can see the stack being helpful for debugging.

Reply:
Added